### ISSUE TITLE
Require more user friendly message if form deleted

### ISSUE THREAD

**Issue by ianholmes** [edina/fieldtrip-gb#234]
If you delete or rename a form (e.g. through DropBox) then press the View/Edit record button in the Authoring Tool for a record that was captured using this now deleted form the user is presented with an error stating:

{"msg": "[404] u'File has been deleted'", "error": 1}

It would be much more helpful if the error stated the problem in plain English, e.g. something like the following:

"This record cannot be displayed as the form (<insert form name>) that was used to capture it has been deleted/renamed."

**Comment by gmh04** [comment]
related https://github.com/edina/fieldtrip-gb/issues/226
